test/create-view-model.ts

```diff
@@ -233,6 +246,35 @@ function tests(model) {
     expect(viewModel.isPropertyDirty("usersInterested")).toBe(false)
     expect(viewModel.isPropertyDirty("unobservedProp")).toBe(false)
 
+    // computed setters shall transparently be called on the view model
+    mobx.runInAction(() => {
+        viewModel.prefixedTitle = "FooBarCoffee"
+    })
+    expect(tr).toBe(
+        "cola:false,interested:Putin,Madonna,Tarzan,Rocky,unobservedProp:new value,usersCount:4"
+    )
+    expect(vr).toBe(
+        "Coffee:false,interested:Putin,Madonna,Tarzan,Rocky,unobservedProp:new value,usersCount:4"
+    )
+    expect(viewModel.title).toBe("Coffee")
+    expect(viewModel.prefixedTitle).toBe("StrongCoffee")
+    expect(viewModel.isDirty).toBe(true)
+    expect(viewModel.isPropertyDirty("title")).toBe(true)
+    expect(viewModel.isPropertyDirty("prefixedTitle")).toBe(false)
+
+    viewModel.submit()
+    expect(tr).toBe(
+        "Coffee:false,interested:Putin,Madonna,Tarzan,Rocky,unobservedProp:new value,usersCount:4"
+    )
+    expect(vr).toBe(
+        "Coffee:false,interested:Putin,Madonna,Tarzan,Rocky,unobservedProp:new value,usersCount:4"
+    )
+    expect(model.title).toBe("Coffee")
```

Comment:
This check is redundant, it is already checked by line 266.

Reply:
True -> fixed